contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -64,6 +64,10 @@ public function addDonation( AddDonationRequest $donationRequest ): AddDonationR
 			$donation->markForModeration();
 		}
 
+		if ( $this->policyValidator->isAutoDeleted( $donationRequest ) ) {
+			$donation->markAsDeleted();
```

Comment:
According to the last comment by @TillWMDE, the donation should not only be deleted but also marked for moderation, right?